content/en/docs/contribute/style/hugo-shortcodes/index.md

```diff
@@ -300,6 +298,9 @@ where `<RELATIVE-PATH>` is the path to the sample file to include, relative to t
 {{%/* code file="configmap/configmaps.yaml" */%}}
 ```
 
+The legacy `{{%/* codenew */%}}` shortcode is being replaced by `{{%/* code */%}}`.
```

Comment:
@sftim, can we put this under the NOTE section, that will look better?

Reply:
I can't find a way to make that work.

Reply:
can we write them under a shell or none code block:

if we use **```none** code then it looks like this:

![none](https://github.com/kubernetes/website/assets/50987400/e7ff7beb-065a-448b-85e6-a9293a869d28)


if we use **```shell** code block then it looks like this:

![shell](https://github.com/kubernetes/website/assets/50987400/c0a623e3-b5f3-4bcf-971f-f34b0bb1552a)

Reply:
If the above seems good then we can change it according to any one of them. IMO using one of them makes context more highlighted. Otherwise, we can go ahead with your changes.

WDYT?

Reply:
I prefer the way I've done it. A code block is the wrong workaround.

Reply:
ok @sftim.